ptu/global.ini

```diff
@@ -73562,11 +73701,11 @@ xenothreat_defend_HH_VH_desc_001=Die Zeit, die die Kopfgeldjäger mit uns versch
 xenothreat_eliminateall_CFP_outpost_title_01=Kill Order: "Citizens" at ~mission(location)
 xenothreat_eliminateall_HH_derelicts_desc_01=Wir haben Informationen erhalten, dass die Kopfgeldjäger versuchen, eine Aktion bei ~mission(location|address) zu starten.\n\nEs scheint, als hätten sie immer noch die irreführende Vorstellung, dass Pyro ihnen gehört. Befreie sie von ihren Wahnvorstellungen und lasse niemanden am Leben. Du wirst nach erfolgreichem Abschluss dieser Mission entschädigt.\n\nKommunikationsspezialist: Engler
 xenothreat_eliminateall_HH_derelicts_title_01=Kill Order: Headhunters at ~mission(location)
-xenothreat_eliminatespecific_desc_001=Die Bemühungen von XenoThreat, zu verhindern, dass externe Einflüsse dieses System infizieren, werden von der "Bürger des Wohlstands"-Aktivistengruppe auf die Probe gestellt.\n\n\nAUFTRAGSDETAILS\n\n   • Zielperson ~mission(TargetName) erledigen\n   • Begib dich zum Einsatzort:\n      > ~mission(Location|Address)\n\n\nEin prominentes Mitglied dieser Gruppe, verbreitet bereits fleißig Lügen und Propaganda an alle, die zuhören. Wenn wir wollen, dass Pyro unabhängig von der UEE und Alien-Einflüssen bleibt, muss jetzt etwas dagegen unternommen werden.\n\nDeshalb gibt XenoThreat einen Tötungsbefehl für ~mission(TargetName|Last) aus. Und in Anbetracht ihrer Bedeutung für die Operationen der "Bürger des Wohlstands" würde es mich nicht überraschen, wenn du dich erst durch eine Menge ihrer fanatischsten Anhänger kämpfen musst, bevor du überhaupt eine Chance bekommst sie zu erwischen. Aber vergiss nicht, ~mission(TargetName|Last) ist dein Ziel, und die Bezahlung hängt allein davon ab, dass du sicherstellst, dass sie beseitigt wird.\n\nKommunikationsspezialist: Engler
+xenothreat_eliminatespecific_desc_001=Die Bemühungen von XenoThreat, zu verhindern, dass externe Einflüsse dieses System infizieren, werden von der "Bürger für Wohlstand"-Aktivistengruppe auf die Probe gestellt.\n\n\nAUFTRAGSDETAILS\n\n   • Zielperson ~mission(TargetName) erledigen\n   • Begib dich zum Einsatzort:\n      > ~mission(Location|Address)\n\n\nEin prominentes Mitglied dieser Gruppe, verbreitet bereits fleißig Lügen und Propaganda an alle, die zuhören. Wenn wir wollen, dass Pyro unabhängig von der UEE und Alien-Einflüssen bleibt, muss jetzt etwas dagegen unternommen werden.\n\nDeshalb gibt XenoThreat einen Tötungsbefehl für ~mission(TargetName|Last) aus. Und in Anbetracht ihrer Bedeutung für die Operationen der "Bürger für Wohlstand" würde es mich nicht überraschen, wenn du dich erst durch eine Menge ihrer fanatischsten Anhänger kämpfen musst, bevor du überhaupt eine Chance bekommst sie zu erwischen. Aber vergiss nicht, ~mission(TargetName|Last) ist dein Ziel, und die Bezahlung hängt allein davon ab, dass du sicherstellst, dass sie beseitigt wird.\n\nKommunikationsspezialist: Engler
```

Comment:
Lassen wir "Bürger für Wohlstand"? Für mich wären die Citizens for Prosperity ein Eigenname uns sollte nicht übersetzt werden

Reply:
Jap :)